appveyor.yml

```diff
@@ -0,0 +1,49 @@
+environment:
+  # There is no need to run the build for all the Python version /
+  # architectures combo as the generated nilearn wheel is the same on all
+  # platforms (universal wheel).
+  # We run the tests on 2 different target platforms for testing purpose only.
+  # We use miniconda versions of Python provided by appveyor windows images
+  matrix:
+    - PYTHON: "C:\\Miniconda"
```

Comment:
Not sure we want to test 32bit python as I said somewhere else (probably the issue associated to this PR)